src/pages/o/[orgId].tsx

```diff
@@ -0,0 +1,39 @@
+import {
+    GetServerSideProps,
+    NextPageContext,
+} from 'next';
+import Link from 'next/link';
+
+export const getServerSideProps : GetServerSideProps = async (context : NextPageContext) => {
+    const { orgId } = context.params;
+
+    const res = await fetch(`http://api.zetk.in/v1/orgs/${orgId}`);
+    const data = await res.json();
```

Comment:
All `await` operations must be enclosed in a `try {} catch {}`, to handle errors that may arise from network issues, missing content or other expected error cases.

For example, try browsing to /o/123456 (which does not exist), and you will see an exception instead of a basic "not found" page. This is an error.